prime-dotnet-webapi/Controllers/EmailsController.cs

```diff
@@ -0,0 +1,39 @@
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+using Prime.Auth;
+using Prime.Services;
+
+namespace Prime.Controllers
+{
+    [Produces("application/json")]
+    [Route("api/emails")]
+    [ApiController]
+    public class EmailsController : ControllerBase
+    {
+        private readonly IEmailService _emailService;
+
+        public EmailsController(IEmailService emailService)
+        {
+            _emailService = emailService;
+        }
+
+        // GET: api/emails/management/statuses
+        /// <summary>
+        /// Update all logged email statuses sent using the CHES email service
+        /// </summary>
+        [HttpGet("management/statuses", Name = nameof(UpdateEmailLogStatuses))]
```

Comment:
I think a POST here would be better, we may actually need/want a GET on these in the future